docs/changelog.md

```diff
@@ -5,6 +5,7 @@ Lines for version numbers should always be formatted as
 with nothing else on the line.
 -->
 * HEAD
+    * [improvement] Added support for Python 3.10
```

Comment:
I'm actually concerned that because we use a Py 3.9 base image, the Py executable used for cloud workers will always be 3.9. We need to follow up and check this.